### PR TITLE
Add BharatPe

### PR DESCRIPTION
**Pull request by eshamahendra**
This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [] You are an employee of the company mentioned and confirm all included details are correct.
- [] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ Name | Website | Region
 [Betable](/company-profiles/betable.md) | https://corp.betable.com/ | USA, Europe
 [BetaPeak](/company-profiles/betapeak.md) | https://betapeak.com/ | Worldwide
 [BetterUp](/company-profiles/betterup.md) | https://www.betterup.com/ | USA
+[BharatPe] |www.bharatpe.com|India
+
```
Comment (KieranRobson) [inline README.md:87]: Please remove this extra line

**Review by KieranRobson — COMMENTED** [review]
Please also add a company profile, an example can be seen at /companyprofiles/example.md

These are required for new companies

File: README.md (hunk 1 above)
Comment (eshamahendra) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -83,8 +83,7 @@ Name | Website | Region
 [Betable](/company-profiles/betable.md) | https://corp.betable.com/ | USA, Europe
 [BetaPeak](/company-profiles/betapeak.md) | https://betapeak.com/ | Worldwide
 [BetterUp](/company-profiles/betterup.md) | https://www.betterup.com/ | USA
-[BharatPe] |www.bharatpe.com|India
-
+[BharatPe] (/company-profiles/bharatpe.md) | https://www.bharatpe.com | India
```
Comment (KieranRobson) [inline README.md:86]: A small typo, please remove the ` ` from the first section

File: company-profiles/bharatpe.md.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## Company size
+
+0 - 1000
+	
+## Remote status
+
+Bharatpe works to provide a professional, hybrid , fun environment at its Gurgaon Office, including healthy snacks; an open, inclusive workspace onsite fitness and trainings for all employees, flexible work schedules.
+
```
Comment (KieranRobson) [inline company-profiles/bharatpe.md.txt:14]: - This file should just be called `company-profiles/bharatpe.md` without the `.txt.`
- There some missing sections, were they missed deliberately?

**Review by KieranRobson — COMMENTED** [review]
Thanks for making those changes however theres a couple more to be made!

**Pull request merged**